src/Route.php

```diff
@@ -15,8 +14,7 @@
 /**
  * Route defines a mapping from URL to callback / name and vice versa.
  */
-#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_CLASS | Attribute::IS_REPEATABLE)]
-class Route implements Stringable
+final class Route implements Stringable
```

Comment:
Maybe we leave it as attribute like `Group`?

Reply:
This class has parameter "action" in constructor that is not needed in attribute.